### PR TITLE
Merge training

### PR DESCRIPTION
**Pull request by astrojuanlu**
Merge all material from https://github.com/kedro-org/kedro-training in a `training` directory, retaining git history thanks to https://github.com/newren/git-filter-repo.

Made one small note to the README on top of the current history.

Addresses https://github.com/kedro-org/kedro-devrel/issues/39.

### CODE REVIEW

**Comment by astrojuanlu** [comment]
Now the key question: should I

- Signoff all the commits myself,
- Manually signoff each commit by their author (sounds like bad idea legally speaking), or
- Give up on retaining the history and squash everything (😢)?

**Comment by astrojuanlu** [comment]
According to https://github.com/git/git/blob/b2c150d3aa82f6583b9aadfecc5f8fa1c74aca09/Documentation/git-commit.txt#L154-L161 ([source](https://stackoverflow.com/a/35238070/554319)):

> The meaning of a signoff depends on the project, but it typically certifies that committer has the rights to submit this work under the same license and agrees to a Developer Certificate of Origin (see https://developercertificate.org/ for more information).

And from https://probot.github.io/apps/dco/ and https://developercertificate.org/,

> By making a contribution to this project, I certify that:
> 
> (a) The contribution was created in whole or in part by me and I have the right to submit it under the open source license indicated in the file; or
> 
> (b) The contribution is based upon previous work that, to the best of my knowledge, is covered under an appropriate open source license and I have the right under that license to submit that work with modifications, whether created in whole or in part by me, under the same open source license (unless I am permitted to submit under a different license), as indicated in the file; or
> 
> (c) The contribution was provided directly to me by some other person who certified (a), (b) or (c) and I have not modified it.

So my understanding is that I can assume under (b) that I can sign-off everything. Happy to learn more otherwise.

File: training/README.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Kedro Training
+
+This repository contains training materials that will teach you how to use [Kedro](https://github.com/kedro-org/kedro). This content is based on the standard [spaceflights tutorial described in the Kedro documentation](https://kedro.readthedocs.io/en/stable/tutorial/spaceflights_tutorial.html).
```
Comment (stichbury) [inline training/README.md:3]:
```suggestion
This repository contains legacy materials used previously for basic [Kedro](https://github.com/kedro-org/kedro) training. The content is based on the standard [spaceflights tutorial described in the Kedro documentation](https://kedro.readthedocs.io/en/stable/tutorial/spaceflights_tutorial.html).
```

Comment (stichbury) [reply training/README.md:3]: I've gone hard on making it clear this is old

File: training/README.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+The training documentation was most recently updated against Kedro 0.17.2 in March 2021.
+
+To get started, navigate to the [training_docs](./training_docs/01_welcome.md) to see what is covered in the training, and how to ensure you get the most out of the time you set aside for it.
```
Comment (stichbury) [inline training/README.md:7]:
```suggestion
To get started, navigate to [training_docs](./training_docs/01_welcome.md) to see what is covered.
```

**Review by stichbury — APPROVED** [review]
LGTM

**Comment by astrojuanlu** [comment]
Continued in gh-5.

**Pull request closed without merging**